docs/modules/ROOT/pages/type-definitions/constraints.adoc

```diff
@@ -59,6 +59,54 @@ type Colour @node(label: "Color", additionalLabels: ["Hue"]) {
 }
 ----
 
+=== `connectOrCreate` mutations
```

Comment:
I don't think this is the correct file for this.

Reply:
Maybe not, the reasoning is because it is part of unique fields only, it would make sense to have it as part of that, I guess mutations/create and /update are better places